tests/cupy_tests/core_tests/test_ndarray_copy_and_view.py

```diff
@@ -335,6 +335,37 @@ def test_copy_host_to_device_view(self):
             dev[:] = host
 
 
+class TestViewDtype:
```

Comment:
It might be a good chance to separate the `view` tests.

Could you rearrange tests in `TestArrayCopyAndView` and this `TestViewDtype` into new classes `TestArrayCopy` and `TestView`?

Reply:
Sure!